doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -587,6 +587,11 @@ Other libraries
     * :c:func:`hw_unique_key_derive_key` function to always return an error code from the library-defined codes.
     * The defined error code names with prefix HW_UNIQUE_KEY_ERR_*.
 
+* :ref:`st25r3911b_nfc_readme` library:
+
+  * Fixed:
+
+    * An issue when :c:func:`st25r3911b_nfca_process` returns error in case when the Rx complete event is received together with FIFO water level event.
```

Comment:
```suggestion
  * Fixed an issue where :c:func:`st25r3911b_nfca_process` function returns an error in case the Rx complete event is received together with FIFO water level event.
```